File: official/guides/customs-guide/golang/buy-international-shipment.go
shipment, err := client.GetShipment("shp_...")
rate := shipment.Rates[0]
shipment, err := client.BuyShipment(shipment.ID, rate, "100.00")

// print PDF link
fmt.Println(shipment.PostageLabel.LabelURL)
// print tracking code
fmt.Println(shipment.TrackingCode)

File: official/guides/customs-guide/python/buy-international-shipment.py
shipment.buy(rate=shipment.lowest_rate(carriers=["USPS"], services=["First"]))

# OR

shipment.buy(rate={"id": "INSERT RATE ID HERE"})

# Print PNG link
print(shipment.postage_label.label_url)
# Print Tracking Code
print(shipment.postage_label.tracking_code)
